core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1744,6 +1744,19 @@ public Map<String, Map<String, Long>> getConnectorRequestsCounts(
     return allCountsMap;
   }
 
+  public MessageSchema getTeamIdFromSchemaNameAndEnvAndTenantId(
+      String schemaTopicName, String envId, int tenantId) {
+    List<MessageSchema> schema =
+        messageSchemaRepo.findAllByTenantIdAndTopicnameAndEnvironment(
+            tenantId, schemaTopicName, envId);
+    if (schema.isEmpty() || schema.size() > 1) {
```

Comment:
wondering if schema.size() > 1 is correct ?

Reply:
Good Call as their may be more then 1 version of a schema that could potentially return null. Will update that.